# SLQ Controller
# Gain matrix, feedforward inputs and timeStamp

float32 			   timeStamp				# time
float32[]			   gainMatrix				# Gain matrix
mpc_input	  		   uff						# feedforward vector